docs/ocis/release_notes.md

```diff
@@ -7,6 +7,12 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
 
+## ownCloud Infinite Scale 1.1.0 Technology Preview
+
+Version 1.1.0 is a hardening and patch release. It ships with version 1.0.2 of ownCloud Web and brings a couple of dependency updates and minor improvements. The minor version increase is needed due to changes in configuration. The documentation has been updated to reflect the changes. Deployments with version 1.0.0 will not break, but the 1.1.0 deployment best practises are not backwards compatible.
```

Comment:
ownCloud Web is at version `1.0.1` in this release. we didn't upgrade to `1.0.2` because that was only a release for fixing the version number in the oc10 marketplace app release.